tests/unit/trace/ext/flask/test_flask_middleware.py

```diff
@@ -117,6 +117,7 @@ def test_init_app_config_stackdriver_exporter(self):
                 'ZIPKIN_EXPORTER_SERVICE_NAME': 'my_service',
                 'ZIPKIN_EXPORTER_HOST_NAME': 'localhost',
                 'ZIPKIN_EXPORTER_PORT': 9411,
+                'ZIPKIN_EXPORTER_PROTOCOL': 'http',
```

Comment:
Shouldn't this not require to be set?

Reply:
None of them are required, they all have defaults in opencensus/trace/exporters/zipkin_exporter.py and opencensus/trace/ext/flask/flask_middleware.py. I thought it was better to follow what was done with the others.

Reply:
Sounds good then.